src/xdocs/property_types.xml

```diff
@@ -928,6 +928,9 @@
         It can be a URL, regular file, or a resource path. It will try loading the path as
```

Comment:
Just saying, i find it odd this sentence didn't change. We have 4 checks, 1 is brand new, but only 3 are mentioned.

Reply:
Added.